insonmnia/miner/overseer.go

```diff
@@ -34,13 +35,22 @@ type Description struct {
 	RestartPolicy container.RestartPolicy
 	Resources     container.Resources
 	Cmd           []string
-	Env           []string
+	Env           map[string]string
 	TaskId        string
 	CommitOnStop  bool
 
 	GPURequired bool
 }
 
+func (d *Description) FormatEnv() []string {
+	vars := make([]string, 0, len(map[string]string(d.Env)))
+	for k, v := range map[string]string(d.Env) {
```

Comment:
Is there any reason to wrap `d.Env` (type `map[string]string`) with another `map[string]string` ? 
Also, it works OK if `d.Env` is nil map.

Reply:
No reason. One day it was a real wrapper, not - it's not.